### ISSUE TITLE
trap does not work when navigating with screen readers using just arrows on Safari 

### ISSUE THREAD

**Issue by julianpinedayyz** [julienw/jquery-trap-input#11]
Tested your examples and the screen reader leaves the trap.  Any way we can solve that?

Tested on Safari latest version with VoiceOver

**Comment by julienw** [comment]
I'm not sure. And I don't have a Mac so I can't really test myself. Is there a way to intercept these keypresses from the page ?

**Comment by julienw** [comment]
Maybe mixing this with ARIA attributes could help ?

**Comment by julienw** [comment]
hey @nico3333fr did you get this issue with your own accessible jquery plugins ?

**Comment by nico3333fr** [comment]
@julienw as far as I know, I didn't have this problem.

Maybe we should ask to @goetsu he will probably be able to help us on this.  (I don't have a mac :-\ )

**Comment by goetsu** [comment]
@miroamarillo safari iOS or osX ?

**Comment by julianpinedayyz** [comment]
@goetsu Safari on OSX latest version.  It seems to be a bug all across operating systems.  It is present on Chrome, JAWS and Windows combo as well. Tab one and start using your arrow keys only and you'll see it present.

**Comment by goetsu** [comment]
I don't have this issue on the dialog demo but I have it on this one https://julienw.github.io/jquery-trap-input/examples/complex.html 
As @nico3333fr script in only about modal I can't compare.
